x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -8,40 +10,276 @@
 
 beta[]
 
-The `elastic-agent run` command provides flags that alter the behavior of an
-agent:
+{agent} provides commands for running the agent and doing common tasks:
 
-`--path.home`::
-The home directory of the {agent}. `path.home` determines the location of the
-configuration files and data directory.
+* <<elastic-agent-enroll-command,enroll>>
+* <<elastic-agent-help-command,help>>
+* <<elastic-agent-inspect-command,inspect>>
+* <<elastic-agent-run-command,run>>
+* <<elastic-agent-version-command,version>>
+
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-enroll-command]]
+== elastic-agent enroll
+
+Enroll the {elastic-agent} in {fleet}.
+
+We recommend that you run this command as the root user because some
+integrations require root privileges to collect sensitive data. This command
+overwrites the `elastic-agent.yml` file in the agent directory.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent enroll <kibana_url> <enrollment_token> [--ca-sha256 <string>]
+                     [--certificate-authorities <string>] [--force] [--help]
+                     [--insecure][global-flags]
+----
+
+[discrete]
+=== Options
+
+`kibana_url`::
+Required. URL of the {kib} endpoint where {fleet} is running.
+
+`enrollment_token`::
+Required. Enrollment token generated by {fleet}. You can use the same
```

Comment:
I'll also change this to say Ingest Manager because I think drawing the distinction between Fleet and Ingest Manager is possibly confusing.